portals/admin/src/main/webapp/source/src/app/components/GatewayEnvironments/AddEditGWEnvironment.jsx

```diff
@@ -514,10 +520,32 @@ function AddEditGWEnvironment(props) {
                         </RadioGroup>
                     </FormControl>
                 )}
+                <FormControl
+                    component='fieldset'
+                    variant='outlined'
+                    margin='dense'
+                    style={{ marginTop: '10px', marginBottom: '10px' }}
+                >
+                    <InputLabel id='demo-simple-select-label'>Type</InputLabel>
+                    <Select
+                        labelId='demo-simple-select-label'
+                        id='demo-simple-select'
+                        value={type}
+                        name='type'
+                        label='Type'
+                        onChange={onChange}
+                        disabled={editMode}
+                    >
+                        <MenuItem value='hybrid'>hybrid</MenuItem>
```

Comment:
Why don't we use capitalized words here?

Reply:
![Screenshot from 2024-02-19 08-30-41](https://github.com/wso2/apim-apps/assets/8374895/7983f2be-a6f0-4147-8475-362b58defe2d)